### ISSUE TITLE
Ruby 2.x compatibility

### ISSUE THREAD

**Issue by benodom** [mheffner/awsam#12]
Placeholder for 2.x compatibility check.

Example: Using Ruby 2.2, there appears to be a silent failure to pull down any ec2 tags from the right_aws api when using assh.

**Comment by adamyonk** [comment]
I'm also seeing this whenever I start a new shell with my system set to use ruby 2.2:

```
Last login: Tue Jan  6 10:01:37 on ttys000
-bash: eval: line 49: syntax error near unexpected token `newline'
-bash: eval: line 49: `keys: {"portal-keypair"=>#<Awsam::Key:0x007f85c412c3b0 @name="portal-keypair", @dir="/Users/adam/.awsam/accts/librato_production/keys/portal-keypair">} export AMAZON_SSH_KEY_NAME="portal-keypair" export AMAZON_SSH_KEY_FILE="/Users/adam/.awsam/accts/librato_production/keys/portal-keypair/key.pem"'
 ✖ adam
 ➔
```

**Comment by mheffner** [comment]
Looks like we may need to upgrade the right_aws -> right_aws_api gem.

**Comment by mheffner** [comment]
Can you give this a shot: https://github.com/mheffner/awsam/pull/14

**Comment by mheffner** [comment]
New PR here: https://github.com/mheffner/awsam/pull/29  cc @adamyonk

**Comment by woqer** [comment]
Also the warning about Digest: `Digest::Digest is deprecated; use Digest` (on Ruby 2.3.0)

**Comment by mheffner** [comment]
Fixed in 0.1.0
